@navikt/core/react/src/table/ExpandableRow.tsx

```diff
@@ -65,17 +65,40 @@ export const ExpandableRow: ExpandableRowType = forwardRef(
   ) => {
     const [internalOpen, setInternalOpen] = useState<boolean>(defaultOpen);
     const id = useId();
+    const expandableButtonRef = useRef<HTMLButtonElement | null>(null);
+    const expandableIconRef = useRef<SVGSVGElement | null>(null);
 
     const isOpen = open ?? internalOpen;
 
+    const expansionHandler = (e) => {
+      onOpenChange?.(!isOpen);
+      if (open === undefined) {
+        setInternalOpen((open) => !open);
+      }
+      e.stopPropagation();
+    };
+
+    const onRowClick = (e) => {
+      if (
+        e.target.nodeName === "TD" ||
+        e.target === expandableButtonRef.current ||
+        e.target === expandableIconRef.current
```

Comment:
Trenger vi disse to sjekkene, siden knappen bruker expansionHandler direkte? 
onRowClick trenger ikke håndtere knappetrykk slik jeg forstår koden.
Da kan vi fjerne de to ref-ene også.

Reply:
Du er rett i at det ikke er nødvendig. Click event'et skal catches av knappen under.
Jeg rydder opp :)